src/models/assetGroupSighting/useAssetGroupSighting.js

```diff
@@ -0,0 +1,13 @@
+import { getAGSQueryKey } from '../../constants/queryKeys';
```

Comment:
IMHO maybe we should start being more consistent with our naming conventions of these custom hooks. Do we include, "Get" in the file name? As in, useGetAssetGroupSighting, or not?

Reply:
this isn't a custom hook, it's a query key getter. all custom hooks should begin with "use" for sure! now whether "get" is the best prefix for query keys... i guess it could also be derive? but for me get is ok too

Reply:
Sorry... iirc, this comment was for that line, but rather for the file itself, which is named useAssetGroupSighting.js...

Reply:
oh yes, we could definitely rename this to `useGetAssetGroupSighting` or `useGetAGS`